pyanaconda/ui/tui/spokes/time_spoke.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, app, data, storage, payload, instclass):
         # so whatever
         self._regions = list(timezone.get_all_regions_and_timezones().keys())
         self._timezones = dict((k, sorted(v)) for k, v in timezone.get_all_regions_and_timezones().items())
-        self._lower_regions = [r.lower() for r in self._timezones]
+        self._lower_regions = [r.lower() for r in self._regions]
```

Comment:
It does make a difference. The order of regions in `self._regions` has to be the same as in `self._lower_regions`, but it was not.